### ISSUE TITLE
How to add validation dataset or record validation loss during the training process?

### ISSUE THREAD

**Issue by 14H034160212** [tatsu-lab/stanford_alpaca#217]
Hi,

I got one question that how to add validation dataset or record validation loss into the weight and bias during the [fine-tuning process](https://github.com/tatsu-lab/stanford_alpaca#fine-tuning)? Thanks in advance.
